scripts/tmfpython.sh

```diff
@@ -132,17 +131,31 @@ tmfpython3 -m methods.inputs.generate_country_raster --jrc /maps/forecol/data/JR
 --output "${output_dir}/${proj}/countries.tif"
 echo "--Country raster created.--"
 
+#Create ecroregion raster
+python3 -m methods.inputs.generate_ecoregion_rasters --jrc /data/jrc/AnnualChange/ \
+--matching /data/tmf/project_boundaries/123/matching-area.geojson \
```

Comment:
Should be `--matching "${output_dir}/${proj}/matching-area.geojson" \`

Reply:
And JRC is wrong